Add new language folders and templates

arthurmatthew opened the pull request:
As requested by issue #3 , I've added a few new folders with included templates in new languages. 

**New languages added:**
- C
- C++
- C#
- Go
- Java
- PHP
- Python

Feel free to remove any of these languages or suggest more, I just selected a few of the popular ones. By the way, it would be great if this was a Hacktoberfest accepted PR!

arthurmatthew committed:
Add new language folders and templates

Dimas-Saputra-Me reviewed (changes requested):
### Feedback
Thank you for your participations, i forgot to mention in issue #3 (already updated now)

So i need to do it in the right order, so in the first hand you need to change index.js to be able to run another programming language first and then you can create another folder and template (just to make it doesn't look messy at first).

### FIX
- [ ] Update index.js to be able to run or compile another programming language

**\~Thank You\~**

arthurmatthew committed:
Add support for new languages

arthurmatthew commented:
Thanks for your feedback!

I've added support for the new folders I've added. I created an object containing a method for executing/compiling each file type. The issue with this is that the user will need to have these applications installed on their computer. However, that will be unavoidable if the goal is to run each language locally. 

I'm happy to make further changes.

Dimas-Saputra-Me reviewed (approved):
I think it is already as good as I expected, all to-do-list were done correctly. Thank you very much for your contributions. 

**\~Thank you\~**
